README.md

```diff
@@ -97,6 +100,18 @@ dojo build -w # start a file-based watch
 dojo build -s -w=memory -m=dev # build to an in-memory file system with HMR
 ```
 
+### Elements
+
+The path for elements to build can be provided using the repeating options `--elements` or `-e`:
+
+```bash
+dojo build -e src/custom-element-child/CustomElementChild -e src/custom-element-parent/CustomElementParent
+```
+
+### Legacy
+
+To build custom elements for legacy environments use the `--legacy` or `-l` flag. For custom elements built with the legacy flag will need to include the polyfill for the [native shim](https://github.com/webcomponents/custom-elements/blob/master/src/native-shim.js).
```

Comment:
The `For` that starts the second sentence should be removed: `Custom elements built with the legacy flag will need to include the polyfill for the native shim.`